packages/database/prisma/migrations/20240927093413_feedbacks/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Feedback` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "FeedbackTypeEnum" AS ENUM ('BUG_REPORT', 'FEATURE_SUGGESTION', 'OTHER');
+
+-- DropForeignKey
+ALTER TABLE "Feedback" DROP CONSTRAINT "Feedback_userId_fkey";
+
+-- DropTable
+DROP TABLE "Feedback";
+
+-- DropEnum
+DROP TYPE "FeedbackType";
+
+-- CreateTable
+CREATE TABLE "feedbacks" (
+    "id" TEXT NOT NULL,
+    "user_id" TEXT NOT NULL,
+    "type" "FeedbackTypeEnum" NOT NULL,
+    "message" TEXT NOT NULL,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "feedbacks_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "feedbacks" ADD CONSTRAINT "feedbacks_user_id_fkey" FOREIGN KEY ("user_id") REFERENCES "users"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
```suggestion
-- Rename userId
ALTER TABLE "Feedback" RENAME COLUMN "userId" TO "user_id";

-- Rename Table
ALTER TABLE "Feedback" RENAME TO "feedbacks";
```
don't trust blindly the sql generation of prisma, especially when you rename things.